packages/puppeteer-core/src/common/bidi/Connection.ts

```diff
@@ -218,7 +225,7 @@ export class Connection extends EventEmitter {
     } else if (isCDPEvent(event)) {
       cdpSessions
         .get(event.params.session)
-        ?.emit(event.params.event, event.params.params);
+        ?.emit(event.params.event, event.params.params[0]);
```

Comment:
Why this change? I think that's not relevant for BiDi+

Reply:
Because that's the correct typing.